test/e2e/login.spec.ts

```diff
@@ -19,7 +21,7 @@ test('Login with User 1 dev redirect', async ({ page }) => {
   await page.getByLabel('Username').fill('user1');
   await page.getByLabel('Password').fill('user1pass');
   await page.getByRole('button', { name: 'Login' }).click();
-  await page.waitForURL('http://localhost:8080/domains');
+  await page.waitForURL('/domains');
   const locator1 = page.locator('#header-user');
   const userInNav = page.getByRole('heading', { name: 'My.Custom.Domain' });
   const domainHeading = page.getByRole('heading', { name: 'Domains' });
```

Comment:
```suggestion
  await page.waitForURL('/domains');
  
  const locator1 = page.locator('#header-user');
  const userInNav = page.getByRole('heading', { name: 'My.Custom.Domain' });
  const domainHeading = page.getByRole('heading', { name: 'Domains' });
  
```
Empty lines to make it consistent with lines 9-14